routers/api/actions/artifacts_chunks.go

```diff
@@ -58,6 +62,32 @@ func saveUploadChunk(st storage.ObjectStorage, ctx *ArtifactContext,
 	return length, nil
 }
 
+func appendUploadChunk(st storage.ObjectStorage, ctx *ArtifactContext,
+	artifact *actions.ActionArtifact,
+	start, contentSize, runID int64,
+) (int64, error) {
+	end := start + contentSize - 1
+	// build chunk store path
+	storagePath := fmt.Sprintf("tmp%d/%d-%d-%d-%d.chunk", runID, runID, artifact.ID, start, end)
```

Comment:
It looks like duplicate code.

Not a blocker, but if there is a chance to reduce duplication, I think it's worth to try.